axelrod/tests/strategies/test_worse_and_worse.py

```diff
@@ -92,18 +92,18 @@ def test_strategy(self):
         self.versus_test(axelrod.Cooperator(), expected_actions = actions)
                           
         actions = [(C, C), (C, C), (C, D),(D, C)]
-        self.versus_test(opponent=axelrod.MockPlayer([C,C,D,C]), expected_actions = actions)
+        self.versus_test(opponent=axelrod.MockPlayer(actions=[C,C,D,C]), expected_actions = actions)
 
         actions = [(C, C)] * 18 + [(C, D), (D, C)]
-        self.versus_test(opponent=axelrod.MockPlayer([C] * 18 + [D, C]), expected_actions = actions)
+        self.versus_test(opponent=axelrod.MockPlayer(actions=[C] * 18 + [D, C]), expected_actions = actions)
 
         # After round 20, strategy follows stochastic behavior given a seed
         actions = [(C, C)] * 20 + [(C, D), (D, C), (C, C), (C, D)] 
-        self.versus_test(opponent=axelrod.MockPlayer([C] * 20 + [D, C, C, D]), expected_actions = actions, 
+        self.versus_test(opponent=axelrod.MockPlayer(actions=[C] * 20 + [D, C, C, D]), expected_actions = actions, 
```

Comment:
(There are some ongoing work on #884 that might fall over if this goes in as is: that's not a huge deal but something to consider.)